src/randoop/operation/ArrayCreation.java

```diff
@@ -9,20 +9,30 @@
 import java.util.List;
 
 import randoop.ExecutionOutcome;
-import randoop.Globals;
 import randoop.NormalExecution;
 import randoop.main.GenInputsAbstract;
-import randoop.sequence.ExecutableSequence;
+import randoop.sequence.Statement;
 import randoop.sequence.Variable;
-import randoop.util.PrimitiveTypes;
-import randoop.util.Reflection;
+import randoop.types.TypeNames;
 
 /**
- * Immutable.
- * Represents a one-dimensional
- * array creation statement, e.g. "int[] x = new int[2] { 3, 7 };"
+ * ArrayCreation is an {@link Operation} representing the construction of a 
+ * one-dimensional array with a given element type and length.
+ * The operation requires a list of elements in an initializer.
+ * For instance, <code>new int[2]</code> is the {@code ArrayCreation} in the 
+ * initialization<br>
+ * <code>int[] x = new int[2] { 3, 7 };</code><br>
+ * with the initializer list as inputs.
+ * <p>
+ * In terms of the notation used for {@link Operation}, an array creation of an 
+ * array of elements of type <i>e</i> and length <i>n</i> has a signature 
```

Comment:
It took me a couple of readings to parse this sentence.  Maybe add commas (even though they are not grammatical) or rephrase?